pkg/proxy/server.go

```diff
@@ -22,7 +23,32 @@ func New(cfg *rest.Config, b bundle.Bundle, rr rewriter.ResourceRewriter) http.H
 	proxyHandler.ModifyResponse = proxyModifyResponse(rr) //nolint:bodyclose // false positive
 
 	r := mux.NewRouter()
+	r.Use(loggingMiddleware)
 	r.Handle("/api/v1/namespaces/{namespace}/pods/{pod}/log", LogsHandler(b))
 	r.PathPrefix("/").Handler(proxyHandler)
 	return r
 }
+
+type LogRecorder struct {
+	http.ResponseWriter
+	status int
+}
+
+func (r *LogRecorder) Write(p []byte) (int, error) {
+	return r.ResponseWriter.Write(p)
+}
+
+func (r *LogRecorder) WriteHeader(status int) {
+	r.status = status
+	r.ResponseWriter.WriteHeader(status)
+}
+
+func loggingMiddleware(next http.Handler) http.Handler {
```

Comment:
Maybe we could use existing the https://pkg.go.dev/github.com/gorilla/handlers#LoggingHandler

Reply:
I've updated the patch to use gorilla handlers, but be aware the gorilla project was archived and is no longer getting any updates.